mathesar_ui/src/sections/table-view/Header.svelte

```diff
@@ -149,11 +180,43 @@
       </Dropdown>
     </div>
   {/each}
+
   <div class="cell" style="width:{70 + DEFAULT_ROW_RIGHT_PADDING}px;left:{
-      columnPosition.get('__row').width + paddingLeft
-    }px;">
-    <div class="add">
-      +
-    </div>
+    columnPosition.get('__row').width + paddingLeft
+  }px;">
+    <Dropdown closeOnInnerClick={false}
+              contentClass="content"
+              bind:isOpen={newColumnDropdownIsOpen}
+              triggerAppearance="plain">
+
+      <svelte:fragment slot="trigger">
+        <span class="name">Add Column</span>
```

Comment:
Instead of text, it would be better to use a plus icon, as per the UX.

Reply:
Done, thanks!

Reply:
https://github.com/pegnott/mathesar/blob/754414d202a29033bd437f2d845269d5885a5831/mathesar_ui/src/sections/table-view/Header.svelte#L229